op-mode-definitions/openconnect.xml.in

```diff
@@ -13,6 +13,53 @@
             </properties>
             <command>${vyos_op_scripts_dir}/openconnect-control.py --action="show_sessions"</command>
           </leafNode>
+          <tagNode name="user">
+            <properties>
+              <help>Show OpenConnect configured user settings</help>
+              <completionHelp>
+                <script>sudo ${vyos_completion_dir}/list_openconnect_users.py</script>
+              </completionHelp>
+            </properties>
+            <children>
+              <node name="otp">
+                <properties>
+                  <help>Show OTP key information</help>
+                </properties>
+                <children>
+                  <leafNode name="full">
+                    <properties>
+                      <help>Show full settings, including QR code and commands for VyOS</help>
+                    </properties>
+                    <command>${vyos_op_scripts_dir}/show_openconnect_otp.py --user="$4" --info="full"</command>
+                  </leafNode>
+                  <leafNode name="key-hex">
+                    <properties>
+                      <help>Show OTP authentication secret in Hex (used in VyOS config)</help>
+                    </properties>
+                    <command>${vyos_op_scripts_dir}/show_openconnect_otp.py --user="$4" --info="key-hex"</command>
+                  </leafNode>
+                  <leafNode name="key-b32">
+                    <properties>
+                      <help>Show OTP authentication secret in Base32 (used in mobile apps)</help>
+                    </properties>
+                    <command>${vyos_op_scripts_dir}/show_openconnect_otp.py --user="$4" --info="key-b32"</command>
+                  </leafNode>
+                  <leafNode name="qrcode">
+                    <properties>
+                      <help>Show OTP authentication QR code</help>
+                    </properties>
+                    <command>${vyos_op_scripts_dir}/show_openconnect_otp.py --user="$4" --info="qrcode"</command>
+                  </leafNode>
+                  <leafNode name="uri">
+                    <properties>
+                      <help>Show OTP authentication otpauth uri</help>
```

Comment:
I believe "URI" should be capitalized everywhere in help strings, like all standard abbreviations.

Reply:
Agreed
Fixed